client/public/frame.html

```diff
@@ -18,16 +18,17 @@
   </script>
   <!--
     JSPM Generator Import Map
-    Edit URL: https://generator.jspm.io/#Y2NhYGCzD80rySzJSU1hSMpM183MK0lNTy1yMNQz0zM1ZEhJTc7MTczRyyp2MDTQM9YzZChKTUwu0U3Jz3UwNNcz0DPCENAvTi0qSy2CiMMUlZZmpjhYAA0wAgA6XlZ2dAA
+    Edit URL: https://generator.jspm.io/#ZcxBDkAwEEDRWQinMQwhlr2EA9BOZKQqqamtq5OIle1L/s8zgOIag4p6djDLUkpQXjgawh47AsdWtsnjehiqsUWCyJPV0u2boQEbrH9QWS8c9O8Hx5Pj61+ckjgzPOPmBszFaRmMAA
```

Comment:
I think there's a bug with `jspm` right now. This URL doesn't actually open anything for me. However, if I navigate to the above URL and then navigate to https://generator.jspm.io, then it shows what the URL should have displayed. :man_shrugging:

Reply:
Seems to work for me. 😅 Tested the link on mobile, different device.

![Screenshot_2022-08-13-02-19-49-55_4d38fce200f96aeac5e860e739312e76](https://user-images.githubusercontent.com/13085980/184459683-2e4534ba-2edb-450a-ba31-cc55ff776e1e.jpg)